lib/compat/wordpress-6.5/blocks.php

```diff
@@ -45,3 +45,82 @@ function gutenberg_register_metadata_attribute( $args ) {
 	return $args;
 }
 add_filter( 'register_block_type_args', 'gutenberg_register_metadata_attribute' );
+
+require_once __DIR__ . '/block-bindings/index.php';
```

Comment:
Usually, there are no requires inside other files rather than `load.php`
Unless you are calling to `wp-includes` with an absolute path.

Example in `class-wp-rest-customizer-nonces.php`
```
/**
	 * Returns the nonce required to request the customizer API endpoint.
	 *
	 * @access public
	 */
	public function get_save_nonce() {
		require_once ABSPATH . 'wp-includes/class-wp-customize-manager.php';
		$wp_customize = new WP_Customize_Manager();
		$nonce        = wp_create_nonce( 'save-customize_' . $wp_customize->get_stylesheet() );
		return array(
			'success'    => true,
			'nonce'      => $nonce,
			'stylesheet' => $wp_customize->get_stylesheet(),
		);
	}
```